### ISSUE TITLE
Allow CORS in preview

### ISSUE THREAD

**Issue by lkovac22** [zoellner/openapi-preview#5]
I am swagger preview against localhost. When executing in browser, all I need is to enable/allow CORS and I am getting results back. When executing in VS code, I am getting "TypeError: Failed to fetch".

Thank you.

**Comment by zoellner** [comment]
the plugin works as expected for me. 
Can you provide a sample openapi file to reproduces this?
And what versions of VSC and openapi-preview are you using?

**Comment by lkovac22** [comment]
Thank you for your response. I cannot post full openapi yaml file, as it is property of my client, but I can easily describe the problem in more details.

In openapi, I am using localhost as a server (for now; it will change, but while developing I would like to keep it this way)

servers:
  - url: http://localhost:8080

Generated curl command looks like this:

curl -X POST "http://localhost:8080/api-path" -H  "accept: application/json" -H  "Content-Type: application/json" -d "{\"prop1\":\"Ver_2\",\"prop2\":40,\"prop3\":122}"

Curl command nicely executes from the command prompt and return back expected result.

When running swagger UI locally in MS Edge, with extension "Allow CORS: Access-Control-Allow-Origin" turned ON, everything works, I am getting back expected result.

However, when executing the same API from VS Code, it gives back:
Server response/Details: TypeError: Failed to fetch

I get the same "TypeError: Failed to fetch" response in swagger UI in MS Edge when I turn extension "Allow CORS: Access-Control-Allow-Origin" OFF. Then Edge console conatins this error:

Access to fetch at 'http://localhost:8080/api-path' from origin 'http://localhost' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

Thank you.

**Comment by zoellner** [comment]
Thank you for the detail. This is a duplicate of #1 then. As mentioned in there, adding a CSP is the recommended default for VSC extensions but I think adding an option to disable it would be good. open to a PR
